themes/openy_themes/openy_rose/scripts/openy_rose.js

```diff
@@ -274,4 +274,30 @@
     }
   };
 
+  // Dynamic font size for Small banner text.
+  Drupal.behaviors.dynamic_font_size_small_banner = {
+    attach: function (context, settings) {
+      // The value that maximum fit in default font size.
+      const headlineMaxLength = 45
+      const  descriptionMaxLength = 325
+
+      let headline = $('.banner--small .field-prgf-headline'),
+          description =  $('.banner--small .field-prgf-description p')
+
+      // Decrease font size if content is too long.
+      if (headline.text().length > headlineMaxLength) {
+        headline.css({
+          fontSize: '22px',
```

Comment:
Let's try to use relative sizes, for example, `vw` - https://css-tricks.com/almanac/properties/f/font-size/